### PR TITLE
Coveralls

### PR DESCRIPTION
**Pull request by octavianCraciun**
Repair travis build error:
 - revert changes on phantomjs version
 - revert chrome

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ module.exports = function (config) {
 
         // start these browsers
         // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-        browsers: ['PhantomJS2', 'Firefox', 'Chrome'],
+        browsers: ['PhantomJS2', 'Firefox'],
```
Comment (cipix2000) [inline karma.conf.js]: I would prefer this list to be ['Phantom', 'Firefox', 'Chrome']

File: package.json [hunk 2]
```diff
@@ -29,9 +29,8 @@
     "karma-coverage": "^1.1.1",
     "karma-coveralls": "^1.1.2",
     "karma-spec-reporter": "0.0.26",
-    "karma-chrome-launcher": "^2.0.0",
     "karma-firefox-launcher": "^1.0.0",
-    "karma-phantomjs-launcher": "^1.0.2",
+    "karma-phantomjs2-launcher": "^0.5.0",
```
Comment (cipix2000) [inline package.json]: this is deprecated. use phantomjs launcher

File: package.json [hunk 3]
```diff
@@ -29,9 +29,8 @@
     "karma-coverage": "^1.1.1",
     "karma-coveralls": "^1.1.2",
     "karma-spec-reporter": "0.0.26",
-    "karma-chrome-launcher": "^2.0.0",
```
Comment (cipix2000) [inline package.json]: I really like Chrome

File: karma.conf.js (hunk 1 above)
Comment (cipix2000) [inline karma.conf.js]: to enable them all just copy a .travis.yml from any of the repositories with coveralls enabled

**Comment by cristiingineru** [comment]
75% - not bad at all

**Pull request merged**